spec/features/blackout_spec.rb

```diff
@@ -41,6 +41,33 @@
     end
   end
 
+  describe 'recurring blackout' do
```

Comment:
These are great! Just to confirm, these currently fail with the old code, right?

Reply:
I actually didn't check that 😅 I'll check out a different branch and run this suite against it.

Reply:
So I've written this a few different ways on master to try and prove that it wasn't passing before, but it keeps passing regardless with `master`. I've tried running something like:

```
  context 'recurring blackout' do                                                  
    context 'with affected reservations' do                                        
      it 'fails' do                                                                
        FactoryGirl.create(:checked_out_reservation,                               
                           due_date: Time.zone.tomorrow)                           
        visit new_recurring_blackout_path                                          
        check_days                                                                 
        fill_in_blackout_data(start_date: Time.new(2018, 10, 24),                  
                              end_date: Time.new(2018, 10, 31))                    
        expect(page).to have_css('.alert-danger', text: /try again/)               
      end                                                                          
    end                                                                            
                                                                                   
    context 'with unaffected reservations' do                                      
      it 'succeeds' do                                                             
        FactoryGirl.create(:overdue_returned_reservation, start_date: Time.new(2018,1,1), due_date: Time.new(2018,1,23))    
        visit new_recurring_blackout_path                                          
        check_days                                                                 
        fill_in_blackout_data(start_date: Time.new(2018, 10, 24),                  
                              end_date: Time.new(2018, 10, 31))                    
        expect(page).to have_css('.alert-success', text: /successfully created./)  
      end                                                                          
                                                                                   
    end                                                                            
  end                                                                              
                                                                                   
  def fill_in_blackout_data(start_date:, end_date:)                                
    select 'Blackout', from: 'Blackout Type'                                       
    fill_in_date(field: 'start', date: start_date)                                 
    fill_in_date(field: 'end', date: end_date)                                     
    fill_in 'Notice', with: 'Foo'                                                  
    click_on 'Create Blackout'                                                     
  end                                                                              
                                                                                   
  def fill_in_date(field:, date:)                                                  
    fill_in "blackout_#{field}_date", with: date                                   
    find(:xpath, "//input[@id='date_#{field}_alt']", visible: :all)                
      .set(date)                                                                   
  end                                                                              
                                                                                
  def check_days                                                                   
    check 'Sun'                                                                    
    check 'Fri'                                                                    
    check 'Sat'                                                                    
  end                                     
```
The same case fails on the GUI running locally:
<img width="1266" alt="screen shot 2018-10-24 at 4 05 09 pm" src="https://user-images.githubusercontent.com/18421583/47458178-ecab6a80-d7a6-11e8-8592-08f49cfbcb69.png">
<img width="1269" alt="screen shot 2018-10-24 at 4 05 19 pm" src="https://user-images.githubusercontent.com/18421583/47458180-ef0dc480-d7a6-11e8-95fd-e63449389797.png">

Reply:
That is pretty weird... is there a reason why that test only has you checking off the weekend dates?

Reply:
I originally had all the days of the week checked, but it wasn't working. I changed it to see if that had any impact and neglected to change it back before posting it here.